nselib/http.lua

```diff
@@ -1640,6 +1644,30 @@ function get(host, port, path, options)
     response, state = lookup_cache("GET", u.host, u.port, u.path, options);
     if ( response == nil ) then
       response = generic_request(u.host, u.port, "GET", u.path, options)
+      if(response.cookies and #response.cookies>0) then 
+        if (options and options.cookies and #options.cookies>0) then
+          --We replace the value of the cookie if same name cookie exists
+          --Else, we append it in the end.
```

Comment:
@nnposter @cldrn please have a look at https://github.com/nmap/nmap/pull/919. I have tried to address all the issues in that PR.